tests/integration/targets/authenticator_maps_test/tasks/main.yml

```diff
@@ -58,7 +58,8 @@
       ansible.builtin.assert:
         that:
           - fail is failed
-          - fail.msg == 'map_type is team but all of the following are missing: team, organization'
+          - "'team' in fail.msg"
```

Comment:
`{'triggers': ['Triggers must be a valid dict'], 'team': ['You must specify a team with the selected map type'], 'organization': ['You must specify an organization with the selected map type'], 'role': ['You must specify a role with the selected map type']}`

This is the error at present.

No way it asseted to 
`map_type is team but all of the following are missing: team, organization`
^prior to september 25th